front-end/cypress/integration/sprint-6-test-184,185,186,187.spec.ts

```diff
@@ -0,0 +1,71 @@
+// @ts-check
+
+import { GenerateContactObject } from '../support/contacts.spec';
+
+var contact_type;
+var contacts = { Individual: {}, Candidate: {}, Committee: {}, Organization: {} };
```

Comment:
convert "var" to "let" and also, since we are using typescript, it is good practice to assign types when possible. For example, the contact_type on line 5 could be written as such for clarity:

let contact_type: string;

Also, we are using the conventional camelCase in the frontend typescript and snake_case in the backend Python.